src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -138,8 +97,8 @@ the value `"hello"`.
 If you use the wrong value type, the build definition will not compile:
 
 ```scala
-lazy val root = (project in file(".")).
-  settings(
+lazy val root = (project in file("."))
+  .settings(
     name := 42  // will not compile
   )
 ```
```

Comment:
Below it says

> `val`s and `def`s must be separated from settings by blank lines.

(a) is this true? if so there needs to be section explaining the blank line thing. And (b) we're not showing settings at the top level (they're all in `.settings(...)`) so this may be confusing.